package/kernel/mac80211/files/lib/wifi/mac80211.sh

```diff
@@ -76,21 +76,27 @@ detect_mac80211() {
 		config_foreach check_mac80211_device wifi-device
 		[ "$found" -gt 0 ] && continue
 
-		mode_band="g"
-		channel="11"
-		htmode=""
-		ht_capab=""
-
-		iw phy "$dev" info | grep -q 'Capabilities:' && htmode=HT20
+		if iw phy "$dev" info | grep -cim1 '58320 MHz'; then
```

Comment:
This will print the number of lines to stdout

Reply:
You are correct, I have reverted to -q version but without brackets and that works.
Tested on LHGG for ad and hAP ac2 for ac.

Reply:
Ah, now I understand what you did in the first place ;-)